gluster-block.spec.in

```diff
@@ -52,6 +52,7 @@ rm -rf ${RPM_BUILD_ROOT}
 %{_sbindir}/gluster-block
 %{_sbindir}/gluster-blockd
 %{_mandir}/man8/gluster-block*.8*
+%dir %{_localstatedir}/log/gluster-block
```

Comment:
@pkalever yes, this is precisely for the server-not-yet-started case!
of course the other PR #29 works around that problem. But I'd rather have the logs
in the file than on stderr...